rudder-web/src/main/scala/com/normation/rudder/web/components/TechniqueEditForm.scala

```diff
@@ -210,7 +212,9 @@ class TechniqueEditForm(
         case e:EmptyBox => NodeSeq.Empty
```

Comment:
There should be a log here or at lower level if the request failed (if it had to be launched)